docs/_data/toc.yml

```diff
@@ -113,196 +119,199 @@
-  url: extensibility
+  url: "#"
   links:
-    - title: "APIs"
-      url: extensibility/api
-    - title: "Adapters ↆ"
-      url: extensibility/adapters
-      children:
-        - title: Compatibility Matrix
-          url: installation/compatibility-matrix
-        - title: App Mesh
-          url: extensibility/adapters/app-mesh
-        - title: Cilium Service Mesh
-          url: extensibility/adapters/cilium
-        - title: Consul
-          url: extensibility/adapters/consul
-        - title: Istio
-          url: extensibility/adapters/istio
-        - title: Kuma
-          url: extensibility/adapters/kuma
-        - title: Linkerd
-          url: extensibility/adapters/linkerd
-        - title: Network Service Mesh
-          url: extensibility/adapters/nsm
-        - title: NGINX Service Mesh
-          url: extensibility/adapters/nginx-sm
-        - title: Nighthawk
-          url: extensibility/adapters/nighthawk
-        - title: Tanzu
-          url: extensibility/adapters/tanzu-sm
-        - title: Traefik Mesh
-          url: extensibility/adapters/traefik-mesh
-    - title: "Integrations"
-      url: extensibility/integrations
-    - title: "Load Generators"
-      url: extensibility/load-generators
-    - title: "Providers"
-      url: extensibility/providers
-    - title: "UI Plugins"
-      url: extensibility/ui
-    - title: "🦾 Extensions ↆ"
+    - title: Integrations
+      url: extensions/integrations
+    - title: Extensions ↆ
       url: extensions
       children:
-        - title: "Plugin: Kanvas"
+        - title: Helm Kanvas Snapshot
+          url: extensions/helm-kanvas-snapshot
+        - title: MeshSync Snapshot
+          url: extensions/kubectl-meshsync-snapshot
+        - title: Kanvas Design Tools ↆ
           url: extensions/kanvas
-          # grandchildren:
-          # - title: "Sharing a Design"
-          #   url: extensions/sharing-a-design
-        - title: "GitOps: Snapshots"
-          url: extensions/snapshot
+          grandchildren:
+            - title: Component Shape Guide
+              url: extensions/component-shape-guide
+            - title: Edges Style Guide
+              url: extensions/edges-shape-guide
+            - title: Import & Export Designs
+              url: extensions/import-export-designs
+            - title: Merging Designs
+              url: extensions/merging-design
+            - title: Publishing Designs
+              url: extensions/publishing-a-design
+            - title: Sharing Designs
+              url: extensions/sharing-a-design
+            - title: Working with Tags
+              url: extensions/working-with-tags
+        - title: Kanvas Snapshot
+          url: extensions/kanvas-snapshot 
 
-      # children:
-      # - title: Building a Remote Provider
-      #   url: extensibility/providers/building-a-remote-provider
 - title: 🧑‍💻 Contributing & Community
   url: project
   links:
-    - title: Community
+    - title: Meshery Community
```

Comment:
How do you reconcile the justification for removing "...for Meshery" from the title of tutorials against the justification for the need to specify that we're talking about the "Meshery" Community?

I understand removing "...for Meshery" from the tutorial titles given that the topic of Meshery is implicitly understood. Adding "Meshery" to the title of Community is helpful because we need to distinguish between communities? If that isn't the case, then the lack of uniformity for inclusion and exclusion of such specificity seems to be unevenly applied. In applying the same test to the title of each page, I advocate for leaving "Community" as-is and shortening the tutorial titles.